section-5/exercises/denysKozlovskyiSection5.js

```diff
@@ -0,0 +1,170 @@
+// Exercise 1:
+// 1. Write a function which will check if a user has a valid ticket ID and does user have 18+ to enter a club.
+// If condition is true then print 'User is allowed to enter the club!' if false then print 'User entrance is not allowed!'
+// Hint: use operator *&&* to check both conditions.
+function checkingUserIdTicket(ticketID, userAge){
+    if (ticketID === 'Yes' && userAge >=18){
+        return console.log("User entrance is allowed!");
+    } else {
+        return console.log("User entrance is not allowed!");
+    }
+}
+checkingUserIdTicket("Yes", 18);
+checkingUserIdTicket("No", 18);
+checkingUserIdTicket("Yes", 17);
+// 2. Create a function using *||* operator to check if user position is *student* OR a *senior* to get a discount to the course.
+function checkingUserPosition(userPosition) {
+    if (userPosition === "student" || userPosition === "senior") {
+        return console.log("Yo have discount");
+    } else if (userPosition === "") {
+        return console.log("we cannot check you");
```

Comment:
What is the reason of this validation?
If you want to add some validation then it would be better to check all cases which may occur ( empty string, undefined, null, etc..) .